### PR TITLE
Fix quick moving resize

### LINKED ISSUE #5: Ruler resizes if moved too quickly
When moving the ruler (via clicking inside the ruler and dragging it), it will resize if the mouse is moved "too quickly".

To replicate: left click and hold in the middle, then rapidly move the mouse, the ruler will begin to resize.

My system: Windows 10 v1607 build 14393.187, 64bit.

This occurred with the pre-built binary in bin/debug

### PR DESCRIPTION
**Pull request by IsaacMorris1980**
This fixes #5 

No C# compiler warnings or errors Visual Studios 2017
No Style Cop Issues used style cop from contributors guide
Simplified tooltip initialization Visual Studios stated object initialization could be improved
Simplified Sub Opacity MenuItems as Visual Studios Stated object initialization could be improved

### CODE REVIEW

File: Source/Ruler/MainForm.cs [hunk 1]
```diff
@@ -118,6 +121,7 @@ private void Init(RulerInfo rulerInfo)
 			this.CreateMenuItems(rulerInfo);
 
 			RulerInfo.CopyInto(rulerInfo, this);
+			this.previouslyLocked = this.isLocked ? true : false;
```
Comment (andrijac) [inline Source/Ruler/MainForm.cs]: `this.isLocked` is already bool value. No need to get explicit bool value through conditional operator.

File: Source/Ruler/MainForm.cs [hunk 2]
```diff
@@ -374,6 +381,7 @@ protected override void OnMouseDoubleClick(MouseEventArgs e)
 
 		protected override void OnMouseDown(MouseEventArgs e)
 		{
+			this.isLocked = this.previouslyLocked ? true : false;
```
Comment (andrijac) [inline Source/Ruler/MainForm.cs]: Same here.

**Comment by andrijac** [comment]
It look good, just change small bool condition.

**Comment by IsaacMorris1980** [comment]
@andrijac I will fix this when I get home tonight

**Pull request merged**